products/cakephp.md

```diff
@@ -7,13 +7,14 @@ permalink: /cakephp
 alternate_urls:
 -   /cake-php
 -   /cake
+versionCommand: php bin/cake.php version
 releasePolicyLink: https://github.com/cakephp/cakephp/wiki
 changelogTemplate: https://github.com/cakephp/cakephp/releases/__LATEST__
+releaseLabel: "__RELEASE_CYCLE__{% if '__CODENAME__' != '' %} '__CODENAME__'{% endif %}"
```

Comment:
Just my opinion, but I find it strange to have codenames for minor versions as those are for major versions.

Reply:
My goal was to get the codenames into the api. I have updated to a new approach with a collapsed table. WDYT?

Reply:
Still not a big fan, but don't see another way if you want that information in the API.